test/core/test_builtin.py

```diff
@@ -0,0 +1,63 @@
+"""
+Test how builtins are loaded
+"""
+
+from mathics.builtin.makeboxes import MakeBoxes
+from mathics.core.builtin import Builtin
+from mathics.core.definitions import Definitions
+
+
+def test_contribute_builtin():
+    """Test for Builtin.contribute."""
+
+    definitions = Definitions()
+    MakeBoxes.context = "System`"
+    MakeBoxes(expression=False).contribute(definitions)
+
+    class TestBuiltin(Builtin):
+        """
+        <dl>
+          <dt>'TestBuiltin'[$x$]
+          <dd>nothing
+        </dl>
+        """
+
+        messages = {
+            "nomsg": "Test message `1`.",
+        }
+
+        def eval_downvalue(self, expr, evaluation):
+            """expr: TestBuiltin[_:Symbol]"""
+            return
+
+        def eval_upvalue(self, expr, x, evaluation):
+            """expr: G[TestBuiltin[x_:Symbol]]"""
+            return
+
+        def format_parm1(self, expr, x, evaluation):
+            """(CustomForm,): expr: F[x_:Symbol]"""
+            return
+
+        def format_parm2(self, expr, x, y, evaluation):
+            """(MakeBoxes, ):expr: G[x_:Symbol,
```

Comment:
@rocky, the example is this one. If the newline is kept, the regular expression matches on the first line, and the rest is discarded, even if the `re.MULTILINE` parameter is given.

Reply:
Got it. Thanks for the information and clarification.